MediaRecorder.html

```diff
@@ -182,11 +182,11 @@
 <section id="blob-event">
     <h2>Blob Event</h2>
     <dl title='interface BlobEvent : Event' class='idl'>
-        <dt>Constructor(DOMString type, BlobEventInit eventInit)</dt>
+        <dt>Constructor(DOMString type, required BlobEventInit eventInitDict)</dt>
```

Comment:
Actually, you can't use the required keyword here, all arguments are required unless they have the optional keyword. The required keyword is only for dictionary members, as you've done below.